cds/config.py

```diff
@@ -477,6 +485,11 @@ def _(x):
     'cds_embed_video',
     'zip',
 ]
+# Wowza server URL for m3u8 playlist generation
+WOWZA_PLAYLIST_URL = ('https://wowzaqaedge.cern.ch/cdseos/Video/'
+                      '{filepath}/{filename}/playlist.m3u8')
+# Thumbnail size
+THUMBNAIL_SIZE = (180, 180)
```

Comment:
Can we add  a more specific name for this var? i.e. ```VIDEO_POSTER_SIZE``` :)  IMHO it might be confused with other thumbnail sizes